File: basic/inline function/python/src/example_1.py
"""
Contexto: Tienes una función llamada calcularAreaCirculo que toma el radio de un círculo como argumento y devuelve el área del círculo. Esta función se considera innecesaria y debe refactorizarse para eliminarla.
"""
def calcular_area_circulo(radio):
    area = 3.14159265359 * (radio ** 2)
    return area

area_circulo = lambda radio: 3.14159265359 * (radio ** 2)

File: basic/inline function/javascript/example1.js
/*
Contexto: Tienes una función llamada calcularIVA que toma el monto de una factura y el porcentaje de impuesto sobre el valor agregado (IVA), y devuelve el monto total incluyendo el IVA. Esta función se considera innecesaria y debe refactorizarse para eliminarla.
*/
function calcularIVA(monto, porcentajeIVA) {
    var iva = monto * (porcentajeIVA / 100);
    var totalConIVA = monto + iva;
    return totalConIVA;
};

const calculoIva = (
    monto, porcentajeIVA) => monto + (monto * (porcentajeIVA / 100));

module.exports = {calcularIVA, calculoIva};